pages/admin/index.tsx

```diff
@@ -104,44 +112,73 @@ const COLUMNS = [
   },
 ];
 
-// Placeholder data
-const DATA = [
-  {
-    name: {
-      name: 'Steve Rogers',
-      id: '36565',
-    },
-    dateReceived: 'Dec 21 2021, 8:30 pm',
-    permitType: 'Permanent',
-    requestType: 'Replacement',
-    status: 'PENDING',
-  },
-  {
-    name: {
-      name: 'Steve Rogers',
-      id: '36565',
-    },
-    dateReceived: 'Dec 21 2021, 8:30 pm',
-    permitType: 'Permanent',
-    requestType: 'Replacement',
-    status: 'INPROGRESS',
-  },
-  {
-    name: {
-      name: 'Steve Rogers',
-      id: '36565',
-    },
-    dateReceived: 'Dec 21 2021, 8:30 pm',
-    permitType: 'Permanent',
-    requestType: 'Replacement',
-    status: 'COMPLETED',
-  },
-];
+// Application data for table
+type ApplicationData = {
+  name: {
+    name: string;
+    rcdUserId: number;
+  };
+  dateReceived: Date;
+  permitType: string;
+  requestType: string;
+  status: ApplicationStatus | undefined;
+};
+
+// Max number of entries in a page
+const PAGE_SIZE = 1;
 
 // Internal home page - view APP requests
 export default function Requests() {
-  const [permitTypeFilter, setPermitTypeFilter] = useControllableState({ defaultValue: 'All' });
-  const [requestTypeFilter, setRequestTypeFilter] = useControllableState({ defaultValue: 'All' });
+  const [statusFilter, setStatusFilter] = useState<ApplicationStatus>();
+  const [permitTypeFilter, setPermitTypeFilter] = useState<PermitType>();
+  const [requestTypeFilter, setRequestTypeFilter] = useState<string>();
+  const [requestsData, setRequestsData] = useState<ApplicationData[]>();
+  const [searchFilter, setSearchFilter] = useState<string>();
+  const [searchInput, setSearchInput] = useState<string>();
+  const [sortOrder, setSortOrder] = useState<SortOptions>([['dateReceived', SortOrder.DESC]]);
+
+  const [pageNumber, setPageNumber] = useState(0);
+  const [recordsCount, setRecordsCount] = useState(0);
+
+  // Make query to applications resolver
+  const { loading, networkStatus } = useQuery<FilterResponse, FilterRequest>(
+    FILTER_APPLICATIONS_QUERY,
+    {
+      variables: {
+        filter: {
+          order: sortOrder,
+          permitType: permitTypeFilter,
+          requestType: requestTypeFilter,
+          status: statusFilter,
+          search: searchFilter,
+          offset: pageNumber * PAGE_SIZE,
+          limit: PAGE_SIZE,
+        },
+      },
+      notifyOnNetworkStatusChange: true,
+      onCompleted: data => {
+        setRequestsData(
+          data.applications.result.map(record => ({
+            name: {
+              name: record.firstName + ' ' + record.lastName,
+              rcdUserId: record.id,
```

Comment:
ah it looks like we're using the database id and not the rcdUserId here, probably need to change that